spec/system/user_index_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+RSpec.describe 'User Index Page', type: :system do
+  before do
+    driven_by(:rack_test)
+    @user1 = User.create!(name: 'anbehindY', photo: 'person.jpg', bio: 'Full-stack web developer')
+    @user2 = User.create!(name: 'picasso', photo: 'person2.jpg', bio: 'Full-stack web developer')
+    @post1 = Post.create!(author: @user1, title: 'First Post', text: 'This is my first post!')
+    @post2 = Post.create!(author: @user2, title: 'first post', text: 'This is my first post!')
+  end
+
+  context 'when the page is loaded' do
+    it 'shows the username of all users' do
```

Comment:
- [ ] Great effort working on your integration spec milestone, but currently i notice 2 of the test cases are not passing and requires your attention. Kindly work on them and be sure all the test cases are passing before you re-submit for review in order to get your project approve. 👍 👍 

Screenshot 👇 

![image](https://github.com/anbehindY/Dev-Blog/assets/78634841/7c4573db-5131-46f4-a63e-7da5c20cd1f4)

Reply:
The new commit is not showing even though I updated the code base and fix the problem. After researching the GitHub issue, it seems to be fixed automatically after some time. So I will just use this as a review request. I hope the new reviewer sees this comment.